packages/aws-lambda-auto-wrap/src/index.js

```diff
@@ -25,6 +25,7 @@ if (!wrappedHandler) {
 }
 
 exports.handler = function instanaAutowrapHandler(event, context, callback) {
+  // NOTE: wondered why this is not protected with a try/catch, because we do not use shimmer
```

Comment:
TODO: Bastian will take a look

Reply:
@basti1302 Did you have a chance to take a look?

Reply:
The wrapped handler ultimately calls the actual Lambda handler, that is, the customer's handler function. If we would wrap this in a try-catch, we would potentially alter the behavior: If the customer's handler function throws an error, it needs to bubble up through here to the AWS Lambda runtime. We must not catch an exception here.

Alternatively, we can theoretically discuss wrapping `loadTargetHandlerFunction` in a try-catch, because that is our own logic. But how would we handle an error? An error in there would mean we were not able to find the actual Lambda handler, which is basically an unrecoverable failure, which is why we throw the respective errors from that function.

In summary: I think the current behavior, that is, not handling any errors here and instead letting errors from `wrappedHandler` bubble up the call stack is the correct behavior.